app/assets/stylesheets/govuk_publishing_components/components/govspeak/_fraction.scss

```diff
@@ -10,7 +10,7 @@
   .fraction {
     sup,
     sub {
-      @include core-14;
+      @include govuk-font($size: 16);
```

Comment:
Is this change intentional?

Reply:
It is not! thanks 👍

Reply:
fixed in b49f80d7 - as well as another small issue